### PR TITLE
layout-catalog

### PR DESCRIPTION
**Pull request by svyatoslavPy**
- [DEMO LINK](https://svyatoslavPy.github.io/layout_catalog/)
- [TEST REPORT LINK](https://svyatoslavPy.github.io/layout_catalog/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,12 +1,314 @@
...
+          alt="logo"
+        >
+
+        <ul class="menu">
+          <li class="menu__item menu__item--active">
+            <a class="menu__link menu__link--special " href="#apple">
+              Apple
+            </a>
+          </li>
+          <li class="menu__item">
```
Comment (Esceype) [inline src/index.html:28]: Add one empty line between sibling blocks inside the same parent block for both of them.
Same for all the similar cases

File: src/styles/blocks/card.scss [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    grid-template-columns: repeat(3, $card-size);
+  }
+
+  @include full-tablet {
+    grid-template-columns: repeat(4, $card-size);
+  }
+}
+
+
+.card__inner {
```
Comment (Esceype) [inline src/styles/blocks/card.scss:22]: Why can't you use sass nesting for this case?

File: src/styles/blocks/card.scss [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+  }
+
+  @include large-tablet {
+    grid-template-columns: repeat(3, $card-size);
+  }
+
+  @include full-tablet {
+    grid-template-columns: repeat(4, $card-size);
+  }
+}
```
Comment (Esceype) [inline src/styles/blocks/card.scss:19]: Move this block to separate file

File: src/styles/main.scss [hunk 4]
```diff
@@ -1,3 +1,37 @@
...
+}
+
+.container {
+  max-width: 1200px;
+  margin: 0 auto;
 }
+
+
+
+
```
Comment (Esceype) [inline src/styles/main.scss:37]:
```suggestion
}
```

**Review by Esceype — CHANGES_REQUESTED** [review]
GJ!
Read the following comments!
Also your tests report link doesn`t work so make redeploy.

File: src/index.html [hunk 5]
```diff
@@ -1,12 +1,343 @@
...
+        <nav class="menu">
+          <ul class="menu__list">
+            <li class="menu__item menu__item--active">
+              <a
+                class="
+                menu__link
+                menu__link--special
+                is-active"
+                href="#apple"
+              >
```
Comment (maxim2310) [inline src/index.html:31]:
```suggestion
              <a
                class=" menu__link menu__link--special is-active"
                href="#apple"
              >
```

File: src/styles/blocks/card.scss [hunk 6]
```diff
@@ -0,0 +1,81 @@
+.card {
+  width: $card-size;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  font-weight: 400;
```
Comment (maxim2310) [inline src/styles/blocks/card.scss:5]: its default
```suggestion
```

File: src/styles/blocks/card.scss [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+      margin-top: 24px;
+    }
+  }
+
+  @include is-hover(transform, scale(100% + 20%));
+
+  &__picture {
+    display: block;
+    width: 160px;
+    height: 135px;
```
Comment (maxim2310) [inline src/styles/blocks/card.scss:26]: check this size

File: src/styles/blocks/card.scss [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+    margin: 0 auto;
+  }
+
+  &__title {
+    width: 166px;
```
Comment (maxim2310) [inline src/styles/blocks/card.scss:31]: block must take all allowed width
```suggestion
```

File: src/styles/blocks/card.scss [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+  }
+
+  &__price {
+    font-weight: 700;
+    font-size: 16px;
+  }
+
+  &__button {
+    border-radius: 5px;
+    width: 166px;
```
Comment (maxim2310) [inline src/styles/blocks/card.scss:65]: same as above

File: src/styles/utils/resets/resets-default.scss [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+@mixin reset-styles-button {
+  border: none;
+  outline: none;
+}
+
+@mixin default-font-family {
+  font-family: $general-font-family;
+  font-weight: 500;
+  font-style: normal;
+  font-size: 12px;
```
Comment (maxim2310) [inline src/styles/utils/resets/resets-default.scss:16]: why do you need mixins for reset, you will not reuse it, looks like overingenering

File: src/styles/utils/flexs/flexs.scss [hunk 11]
```diff
@@ -0,0 +1,11 @@
+@mixin flex-space-between {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
```
Comment (maxim2310) [inline src/styles/utils/flexs/flexs.scss:5]: logically it also mixin so or move it to mixin file directly or create separate file for all mixins and move it there

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Good job! Let's improve your code a bit)

looks like you have a mistakes in card sizes, check carefully
image

**Review by maxim2310 — CHANGES_REQUESTED** [review]
.

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Yor demo is broken, please fix it and make request review again, also check test and fix indent in your card

File: src/index.html [hunk 12]
```diff
@@ -1,12 +1,341 @@
...
+          src="images/logo.png"
+          alt="logo"
+        >
+
+        <nav class="menu">
+          <ul class="menu__list">
+            <li class="menu__item menu__item--active">
+              <a
+                class="
+                menu__link menu__link--special is-active"
```
Comment (etojeDenys) [inline src/index.html:27]:
```suggestion
                class="menu__link menu__link--special is-active"
```

**Review by etojeDenys — APPROVED** [review]
lgtm

**Pull request closed without merging**